src/otx/v2/configs/detection/otx_mmdet_default.yaml

```diff
@@ -0,0 +1,163 @@
+# Basic Information for Config
+framework: mmdet
+data:
+  task: Detection
+
+  train_type: Incremental
+
+model:
+  name: otx_mobilenetv2_atss
```

Comment:
`train_type` is included in `data` config?

Reply:
I guess it's because auto runner. Maybe we can discuss about this in the execution forum. Unfortunately I don't have much background of this.

Reply:
Alright.